node/silkworm/stagedsync/stage_history_index.cpp

```diff
@@ -239,7 +239,6 @@ StageResult history_index_prune(TransactionManager& txn, const std::filesystem::
                 // generates new key
                 Bytes new_key(key.size(), '\0');
                 std::memcpy(&new_key[0], key.data(), key.size());
-                endian::store_big_u32(&new_key[new_key.size() - 4], UINT32_MAX);
                 // replace with new index
                 etl::Entry entry{new_key, new_bitmap};
```

Comment:
`new_key` is the same as `key`, and you don't actually need it. You can do something like
```
etl::Entry entry{Bytes{key}, new_bitmap};
```

Reply:
ops, forgot to git add it